moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -327,39 +299,37 @@ const ProjectFundingTable = () => {
     }, {});
   };
 
-  /**
-   * Component for dropdown select using a lookup table as options
-   * @param {*} props
-   * @returns {React component}
-   */
-  const LookupSelectComponent = (props) => (
-    <Select
-      style={{ minWidth: "8em" }}
+/**
+ * Component for autocomplete using a lookup table as options
+ * @param {*} props
+ * @returns {React component}
+ */
+  const LookupAutocompleteComponent = (props) => (
```

Comment:
this is great! any reason why you put `LookupSelectComponent` in it's own file and not `LookupAutocompleteComponent` - ?